.github/workflows/on-tag.yml

```diff
@@ -11,11 +11,11 @@ permissions:
 
 jobs:
   test:
-    uses: ./.github/workflows/reusable/build-and-test.yml
+    uses: ./.github/workflows/build-and-test.yml
```

Comment:
Can be squashed with https://github.com/bitcoin-dev-project/sim-ln/pull/149/commits/046ee5d978b1e115df19494c353212ab0c54ab22 👍

Reply:
I just squashed the lot